src/openrct2-ui/windows/Options.cpp

```diff
@@ -251,28 +251,26 @@ static Widget window_options_display_widgets[] = {
     MakeWidget        ({ 11, 206}, {280,  12}, WindowWidgetType::Checkbox,     WindowColour::Secondary, STR_DISABLE_SCREENSAVER,               STR_DISABLE_SCREENSAVER_TIP              ), // Disable screensaver
     kWidgetsEnd,
 };
-
```

Comment:
Please reinstate the blank line between the widget groups. Makes it easier to read.